doc/user/content/connect-sources/cdc-postgres-direct.md

```diff
@@ -11,44 +11,96 @@ menu:
     weight: 5
 ---
 
-Change Data Capture (CDC) allows you to track and propagate changes in a Postgres database to downstream consumers based on its Write-Ahead Log (WAL).
+This page shows you how to stream data directly from a PostgreSQL database to Materialize using PostgreSQL's [logical replication](https://www.postgresql.org/docs/10/logical-replication.html) protocol.
 
-This guide shows you how to propagate CDC data directly from [Postgres](/sql/create-source/postgres) to Materialize using Postgres’ native replication protocol.
+## Before you begin
 
-## Database setup
+- Make sure you are running PostgreSQL 10 or higher.
 
-**Minimum requirements:** PostgreSQL 10+
+- Make sure there are no schema changes planned or in progress on the tables that you want to replicate.
 
-Before creating a source in Materialize, you need to ensure that the upstream database is configured to support [logical replication](https://www.postgresql.org/docs/10/logical-replication.html).
+  If Materialize detects a schema change on a table it is replicating, Materialize will [log an error](/manage/troubleshooting/#why-isnt-my-source-ingesting-data) and stop all replication from the PostgreSQL database. To handle schema changes, you will need to redo the integration. See [Handling schema changes](#handling-schema-changes) for more details.
```

Comment:
This has subtly changed, as per these docs:
https://materialize.com/docs/sql/create-source/postgres/#schema-changes

I might just summarize this as "review the known limitations" or make them a snippet and inline them here.